model_path: /home/tp3/autoware_data/bev_fusion
model_name: resnet50
precision: 1 # NonePrecision = 0, Float16 = 1, Int8 = 2
normalization:
  image_width: 1600
  image_height: 900
  output_width: 704
  output_height: 256
  num_camera: 6
  resize_lim: 0.48
  interpolation: 1  # Nearest = 0, Bilinear = 1
  mean: [0.485, 0.456, 0.406]
  std: [0.229, 0.224, 0.225]
  method: 1 # Nothing = 0, MeanStd = 1, AlphaBeta = 2 

voxelization:
  min_range: [-54.0, -54.0, -5.0]
  max_range: [54.0, 54.0, 3.0]
  voxel_size: [0.075, 0.075, 0.2]
  max_points_per_voxel: 10
  max_points: 300000
  max_voxels: 160000
  num_feature: 5
  
geometry:
  xbound: [-54.0, 54.0, 0.3]
  ybound: [-54.0, 54.0, 0.3]
  zbound: [-10.0, 10.0, 20.0]
  dbound: [1.0, 60.0, 0.5]
  image_width: 704
  image_height: 256
  feat_width: 88
  feat_height: 32
  num_camera: 6
  geometry_dim: [360, 360, 80]   

transbbox:
  out_size_factor: 8
  pc_range: [-54.0, -54.0]
  post_center_range_start: [-61.2, 61.2, -10.0] 
  post_center_range_end: [61.2, 61.2, 10.0]
  voxel_size: [0.075, 0.075]
  confidence_threshold: 0.12
  sorted_bboxes: True